src/components/QuizDisplay.tsx

```diff
@@ -1,112 +1,76 @@
-import { useRef, useState } from "react";
+import { useState } from "react";
 import SingleQuestion from "./SingleQuestion";
-import { IAnswerInfo, IQuestionInfo, QuizStatusType } from "../types";
-import Report from "./Report";
-
-interface QuizDisplayProps {
-  allQuestions: IQuestionInfo[];
-  quizStatus: QuizStatusType;
-  setQuizStatus: React.Dispatch<React.SetStateAction<QuizStatusType>>;
-}
+import { IQuizDisplayProps, IAnswerInfo } from "../types";
+import ReturnToQuizSetupBtn from "./ReturnToQuizSetupBtn";
 
 const QuizDisplay = ({
   allQuestions,
-  quizStatus,
   setQuizStatus,
-}: QuizDisplayProps) => {
+  noOfCorrectAnswers,
+  handleReturnToQuizSetup,
+}: IQuizDisplayProps) => {
   const [selectedAnswer, setSelectedAnswer] = useState<number | null>(null);
   const [allAnswers, setAllAnswers] = useState<IAnswerInfo[]>([]);
   const [answerIsSubmitted, setAnswerIsSubmitted] = useState<boolean>(false);
```

Comment:
Prefer boolean naming convension, ref: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig#:~:text=It%20behaves%20the%20same%20and,on%20simplifying%20writing%20software%20here.
```suggestion
const [isAnswerSubmitted, setIsAnswerSubmitted] = useState<boolean>(false);
```

Reply:
So do I, idk how I didn't spot this myself, I've updated it now